<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <resources><include location="png/icons.qrc"/></resources><author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>QG_LineOptions</class>
  <widget class="QWidget" name="QG_LineOptions">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>1</y>
        <width>200</width>
        <height>22</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy>
        <hsizetype>4</hsizetype>
        <vsizetype>5</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="minimumSize">
      <size>
        <width>200</width>
        <height>22</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>280</width>
        <height>22</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Line Options</string>
    </property>
    <layout class="QHBoxLayout">
      <property name="margin">
        <number>1</number>
      </property>
      <item>
        <widget class="QToolButton" name="bClose">
          <property name="text">
            <string>Close</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QToolButton" name="bUndo">
          <property name="text">
            <string>Undo</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="Line" name="sep1">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="frameShape">
            <enum>QFrame::VLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <includes>
    <include location="local">rs_line.h</include>
    <include location="local">rs_settings.h</include>
    <include location="local">rs_actiondrawline.h</include>
  </includes>
  <connections>
    <connection>
      <sender>bClose</sender>
      <signal>clicked()</signal>
      <receiver>QG_LineOptions</receiver>
      <slot>close()</slot>
    </connection>
    <connection>
      <sender>bUndo</sender>
      <signal>clicked()</signal>
      <receiver>QG_LineOptions</receiver>
      <slot>undo()</slot>
    </connection>
  </connections>
</ui>
